### PR TITLE
70859 contact details unit and int tests

### PR DESCRIPTION
**Pull request by will-hope98**

### CODE REVIEW

File: src/EA.Weee.RequestHandlers/Organisations/DirectRegistrants/CompleteOrganisationTransactionHandler.cs [hunk 1]
```diff
@@ -64,7 +64,19 @@ public async Task<Guid> HandleAsync(CompleteOrganisationTransaction request)
 
                     var brandName = await CreateAndAddBrandName(organisationTransactionData);
 
-                    var directRegistrant = DirectRegistrant.CreateDirectRegistrant(organisation, brandName);
+                    var contactDetails = new Contact(organisationTransactionData.ContactDetailsViewModel.FirstName, organisationTransactionData.ContactDetailsViewModel.LastName,
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.RequestHandlers/Organisations/DirectRegistrants/CompleteOrganisationTransactionHandler.cs:67]: @will-hope98 Just to tidy the code can we move the creation of the objects into local private methods similar to how the others are created

File: src/EA.Weee.Web/Services/OrganisationTransactionService.cs [hunk 2]
```diff
@@ -59,7 +59,7 @@ public async Task CaptureData<T>(string accessToken, T model) where T : class
                         transaction.AuthorisedRepresentative = authorisedRepresentativeViewModel.SelectedValue.GetValueFromDisplayName<YesNoType>();
                         break;
                     case ContactDetailsViewModel contactDetailsViewModel:
-                        transaction.ContactDetails = contactDetailsViewModel;
+                        transaction.ContactDetailsViewModel = contactDetailsViewModel;
```
Comment (JohnPadgettCivica) [inline src/EA.Weee.Web/Services/OrganisationTransactionService.cs:62]: @will-hope98 Does this require the OrganisationTransactionService unit tests to be updated?

**Comment by JohnPadgettCivica** [comment]
@will-hope98  looks like some unit tests are failing https://sfwjenkins01.main.sfwltd.co.uk/job/WEEE/job/WEEE%20CI/381/

File: src/EA.Weee.Web/Services/OrganisationTransactionService.cs (hunk 2 above)
Comment (will-hope98) [reply src/EA.Weee.Web/Services/OrganisationTransactionService.cs:62]: Yep added a test to cover that

File: src/EA.Weee.RequestHandlers/Organisations/DirectRegistrants/CompleteOrganisationTransactionHandler.cs (hunk 1 above)
Comment (will-hope98) [reply src/EA.Weee.RequestHandlers/Organisations/DirectRegistrants/CompleteOrganisationTransactionHandler.cs]: Done

**Pull request merged**